ooyala_player/ooyala_player.py

```diff
@@ -170,14 +149,17 @@ def student_view(self, context):
         """
         Player view, displayed to the student
         """
-        # For lightchild xblock, pass skin resource URL, otherwise use our custom handler method
+        # Skin file path according to block type
```

Comment:
if skin file is same for both block types than i think we can remove this if block